### PR TITLE
adds arquillian pact 1.0.0.Alpha2 announcement

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Will push it once in central. Thanks.

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+author: lordofthejars
+---
+
+p. Arquillian Pact is an extension that provides an integration between Arquillian and "Pact":https://docs.pact.io/.
+
+h3 What's new
+
+p. <i class="icon-star"></i> Maven Loader
+
+p. Now you can use any Maven artifact to store contracts (pacts files). Then provider will resolve given artifact, uncompress it and execute the contracts found there.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > any Maven artifact

Maybe : "your own maven artifact"?

Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > uncompress it and execute the contracts found there.

This is implementation detail - we can just say that provider will resolve the artifact and use the files included there.

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+p. To use it you need to import dependency:
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: >  you need to import dependency

"simply import following dependency" ?

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+bc(prettify).. <dependency>
+  <groupId>org.arquillian.pact</groupId>
+  <artifactId>arquillian-pact-provider-maven-loader</artifactId>
+</dependency>
+
+p. For example, this is how you can get contracts from a jar file stored at Maven repository. It is important to note that Arquillian Pact always takes the latest version of jar in case of not specifying one concrete version.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > stored at Maven repository.

"stored in"?

Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > latest version of jar in case of not specifying one concrete version.

"latest version of **a** jar, when no concrete version is specified." ?

Question: latest what? release? snapshot?

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  <groupId>org.arquillian.pact</groupId>
+  <artifactId>arquillian-pact-provider-git-loader</artifactId>
+</dependency>
+
+p. For example, this is how you can get contracts form a public Git repository:
+
+bc(prettify).. @PactGit("https://github.com/lordofthejars/starwarspact.git")
+public class GitLoaderTest {}
+
+p. Check __PactGit__ annotations to see the options for configuring authentication or set a concrete directory inside Git repo.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > Check __PactGit__ annotation
I would link to the code if there is a javadoc

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+</dependency>
+
+p. For example, this is how you can get contracts from a jar file stored at Maven repository. It is important to note that Arquillian Pact always takes the latest version of jar in case of not specifying one concrete version.
+
+bc(prettify).. @PactMavenDependency(value = "org.superbiz:contract:[1.0,]")
+public class MavenLoaderTest {}
+
+p. <i class="icon-star"></i> Git Loader
+
+p. Now you can get contracts from Git repository. The provider will clone/pull the repo and execute the contracts found there.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: This opens up in exactly the same way as paragraph about maven. I was thinking we can rephrase it to

"You can also fetch your contracts from Git repository"

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+p. For example, this is how you can get contracts form a public Git repository:
+
+bc(prettify).. @PactGit("https://github.com/lordofthejars/starwarspact.git")
+public class GitLoaderTest {}
+
+p. Check __PactGit__ annotations to see the options for configuring authentication or set a concrete directory inside Git repo.
+
+p. <i class="icon-star"></i> Output Directory
+
+p. Now in consumer you can use __pactReportDir__ configuration property to set where contracts are generated.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: As with the previous paragraph - maybe small rephrase to break the scheme?

When writing consumer tests you can now use __pactReportDir__ configuration property to define where contracts are stored.

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+For example, this is how you can generate contracts into target/contracts directory.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: generate > save ?

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+bc(prettify).. <extension qualifier="pact-consumer">
+  <property name="pactReportDir">target/contracts</property>
+</extension>
+
+p. <i class="icon-star"></i> Enrichments
+
+p. It is not mandatory to enrich your test with current consumer and current request response interaction. You can use two new Target methods for such:
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: After reading it I don't really understand how and for what it is used for. We didn't explain annotations you refer to in the first post. Maybe we can elaborate here on both and highlight the differences?

File: blog/arquillian-pact-1.0.0.Alpha2.textile [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+bc(prettify).. @ArquillianResource
+Target target;
+
+@ArquillianResource
+URL webapp;
+
+target.testInteraction(webapp);
+
+p. You can still use previous methods where consumer and interaction where required, but in most of the cases you can relay on these methods.
```
Comment (bartoszmajsak) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: > previous methods **where** consumer and interaction where required

Is this first **where** required in this sentence? Sounds odd.

File: blog/arquillian-pact-1.0.0.Alpha2.textile (hunk 3 above)
Comment (lordofthejars) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: This depends on Maven, but I think it will be the released ones, although Maven document does not specify http://maven.apache.org/enforcer/enforcer-rules/versionRanges.html

File: blog/arquillian-pact-1.0.0.Alpha2.textile (hunk 7 above)
Comment (lordofthejars) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: Well Pact generates the contract files from stub http definition. So although it sounds weird if we want to be strict Pact generates the contracts from code.

File: blog/arquillian-pact-1.0.0.Alpha2.textile (hunk 8 above)
Comment (lordofthejars) [inline blog/arquillian-pact-1.0.0.Alpha2.textile]: Well in first post was mandatory to use them, and the example of the announcement used that, what I am going to do is rephrase this
